XMLProjectIS/Pages/Index.cshtml

```diff
@@ -2,46 +2,12 @@
 @model IndexModel
 @{
     ViewData["Title"] = "Home page";
-    MovieDetails.Item[] movies = (MovieDetails.Item[])ViewData["MovieDetails"];
+    MovieDetails.Item[] moviesDetails = (MovieDetails.Item[])ViewData["MovieDetails"];
 }
+<head>
+   <link rel="stylesheet" href="~/css/StyleSheet.css">
+    </head>
```

Comment:
Good, with a footnote: the head section already exists in _Layout.cshtml, which includes this page.  You can't duplicate the head section, and head should be a child of html.  Best to add the CSS to the _Layout.cshtml.